.github/workflows/lint.yml

```diff
@@ -4,7 +4,7 @@ on:
   workflow_dispatch:
   schedule:
     # every 3 days
-    - cron:  '0 2 */3 * 1'
+    - cron:  '0 2 */3 * *'
```

Comment:
Sorry, I am confused by the change and branch name. 

I'm reading this as running on the third day of the month if it's a Monday originally, and the change as running on every third day of month regardless (which could mean on Sundays).

Reply:
Yea it's supposed to run on some interval. And it's not currently running at all. It's unclear to me why, I'm honestly just hoping this may get it going.

The github editor actually has good support for this

![image](https://user-images.githubusercontent.com/4874123/207157913-5a26af88-225f-4281-b289-af4607d734b9.png)

and 

![image](https://user-images.githubusercontent.com/4874123/207158072-7920f79a-9b76-4a6f-9704-f0ff735d656a.png)

I'm hoping that last bit - `and every monday` may kick start it?

Reply:
Haha, yeah I know what you mean. Ok, I'm good with it just to get it going again hopefully and adjusting if needed.